src/components/Results/Study.tsx

```diff
@@ -100,7 +101,12 @@ const Study = ({ entry, handleSaveStudy, isStudySaved, closestFacility }: StudyP
           </Stack>
 
           <Stack p={2} sx={{ backgroundColor: 'common.grayLighter' }}>
-            <StudyDetailsButton icon={<LaunchIcon />} text="More info" />
+            <StudyDetailsButton
+              icon={<LaunchIcon />}
+              text="More info"
+              target="_blank"
+              href={'https://www.clinicaltrials.gov/ct2/show/' + studyProps.trialId}
```

Comment:
Hey Joyce! When you get a chance can you rebase off of develop? This line should then change to `entry.trialId` as  the resulting prop should have already been converted into `StudyDetailProps`

Reply:
Sure, I'll make a commit to rebase off develop soon.

Reply:
Done!